www/templates/intro/consent.html

```diff
@@ -1,113 +1,105 @@
+miniscule but undeniably present risk regarding a data breach, but we are taking precautions to minimize
+this risk.
+
+<h3>Confidentiality</h3>
+Your study data will be handled as confidentially as possible. If the results of this study are published
+or presented, individual names and other personally identifiable information will not be used. To minimize
+risks to confidentiality, we will store study data on a secure, encrypted E-Mission platform server. The
+server can only be accessed by the researchers and nobody else.
 
 <h3>Your rights</h3>
-<b><i>Participation in research is completely voluntary</i></b>.  You have the
-right to decline to participate or to withdraw at any point in this study
-without penalty or loss of benefits to which you are otherwise entitled.
+<em>Participation in research is completely voluntary.</em> You have the right to decline to participate
+or to withdraw at any point in this study without penalty or loss of benefits to which you are otherwise
+entitled.
+
 
 <h3>Questions</h3>
 If you have any questions about this research, please feel free to contact us.
-For the quickest response, you can send email to the entire team at <a
-href="mailto:e-mission@lists.eecs.berkeley.edu">e-mission@lists.eecs.berkeley.edu</a>.
-If you want to contact only the lead researcher or the PI, you can use their
-profiles above.
+For the quickest response, you can send email to the entire team at
+<a href='emissionberkeley@gmail.com'>emissionberkeley@gmail.com</a>.
+
+
+<h3>Consent</h3>
 
 If you have any questions about your rights or treatment as a research
 participant using this platform, please contact the University of California at
 Berkeley's Committee for Protection of Human Subjects at 510-642-7461, or
 e-mail <a href="mailto:subjects@berkeley.edu">subjects@berkeley.edu</a>.
 
-If you agree to contribute your data to the platform, please click "I agree"
-below to proceed. If you do not wish to contribute your data to the platform,
-please click "I disagree" below to return to the aggregate views. This notice
-is also available <a
-href="https://e-mission.eecs.berkeley.edu/consent">online</a> in case you need
-to read it again in the future.
+Please select the button below to indicate that you have read and agree to this consent document,
+and want to participate in our study.
 
 <div class="row" style="margin-top: 20px;">
-  <div class="col" style="margin-right: 5px">
-    <button class="button button-block button-assertive button-outline" ng-click="disagree()" id="consent-button">I Refuse</button>
-  </div>
-  <div class="col" style="margin-left: 5px">
-    <button class="button button-block button-balanced button-outline" ng-click="agree()" id="consent-button">I Agree</button>
+  <div class="col">
+    <button class="button button-block button-balanced button-outline" ng-click="agree()" id="consent-button">Agree</button>
```

Comment:
IRB didn't require an "I disagree" button? They were very picky when I first filed the protocol.

Reply:
No, we took out the "releasing data after a year" thing